app/assets/javascripts/bootstrap.js

```diff
@@ -0,0 +1,4 @@
+jQuery(function () {
```

Comment:
Should be changed to, for not relying on the global alias for jQuery:
jQuery(function ( $ ) {

reference: http://api.jquery.com/jquery/#jQuery3